tests/DCPS/ParticipantLocationTopic/ParticipantLocationTest.cpp

```diff
@@ -359,6 +362,27 @@ int ACE_TMAIN(int argc, ACE_TCHAR *argv[])
     // Call on_data_available in case there are samples which are waiting
     sub_listener->on_data_available(sub_loc_dr);
 
+    DDS::DataReader_var sub_part_dr = sub_bit_subscriber->lookup_datareader(OpenDDS::DCPS::BUILT_IN_PARTICIPANT_TOPIC);
+    if (!sub_loc_dr) {
+      ACE_ERROR((LM_ERROR,
+                 ACE_TEXT("%N:%l main()")
+                 ACE_TEXT(" ERROR: Could not get %C DataReader\n"),
+                 OpenDDS::DCPS::BUILT_IN_PARTICIPANT_TOPIC));
+      ACE_OS::exit(EXIT_FAILURE);
```

Comment:
These could be returns?

Reply:
Sure, I was just keeping it the same as what the test was already doing for the other BIT reader.